contrib/debian/examples/bitcoin.conf

```diff
@@ -95,26 +95,21 @@
 # running on another host using this option:
 #rpcconnect=127.0.0.1
 
-# Transaction Fee Changes in 0.10.0
+# Wallet options
 
-# Send transactions as zero-fee transactions if possible (default: 0)
-#sendfreetransactions=0
-
-# Create transactions that have enough fees (or priority) so they are likely to begin confirmation within n blocks (default: 1).
+# Create transactions that have enough fees so they are likely to begin confirmation within n blocks.
 # This setting is over-ridden by the -paytxfee option.
 #txconfirmtarget=n
 
+# Pay a transaction fee every time you send bitcoins.
+#paytxfee=0.000x
```

Comment:
All fees exposed via the user interface are BTC per kB (or mBTC per Byte). I don't think it adds value to mention it everywhere.